src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int sideA;
+    private int sideB;
+    private int sideEqual;
```

Comment:
I cant understand what do these fields mean

let's use `height` and `middleLine`, or `height`, `firstBase` and `secondBase`

Reply:
done